2.14/2.14.1/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 2.14.1
+Напишите if..else, соответствующий следующему switch:
+
+switch (browser) {
+case 'Edge':
+alert( "You've got the Edge!" );
+break;
+
+case 'Chrome':
+case 'Firefox':
+case 'Safari':
+case 'Opera':
+alert( 'Okay we support these browsers too' );
+break;
+
+default:
+alert( 'We hope that this page looks ok!' );
+}
+<script>
+	let browser = 'Chrome';
+
+	if (browser === 'Edge') {
+		alert( "You've got the Edge!" );
+	} else if (browser === 'Chrome' || browser === 'Firefox' || browser === 'Safari' || browser === 'Opera') {
```

Comment:
it's good but just rewrite in another way just to know that's possible as well. Try to rewrite these `if`s in order to have separate blocks like this:
`
if { ... } // the first if

if { ... } // the second if
`